test/alternator/suite.yaml

```diff
@@ -1,6 +1,5 @@
 type: Python
 pool_size: 4
-prepare_cql: INSERT INTO system_auth_v2.roles (role, salted_hash) VALUES ('alternator', 'secret_pass')
```

Comment:
Why did we have this command in multiple places...

Reply:
This is exactly what my patch removes. **Before** the patch, we could only run the Alternator tests on a cluster that had been specially prepared by adding a role called "alternator". This is why the two scripts that started Scylla for the Alternator tests - test/alternator/run and test/alternator/suite.yaml (the latter was just added a few days ago, and is used when running through test.py), and also the README.md explaining how to run Scylla manually for these tests - all needed to contain this silly preparation code
After my patch, there is no more need for this preparation code. We don't need to add a new "alternator" role - we have a "cassandra" role, which works just fine. In fact, our CQL tests anyway assume this role exists, and use it (they wouldn't be able to connect to CQL without it!), so it's not a new assumption in our tests.

Reply:
Yes, I know what you meant, and I explained why we had (in past tense) all these copies of that code.
And the nice thing is that we no longer have them and instead have all the mess in one place (conftest.py) ;-)